topics/templates/document-attributes-product.adoc

```diff
@@ -10,10 +10,10 @@
 :on-prem:
 
 :project_name_full: Red Hat Single Sign-On
-:project_version_base: 7.5
-:project_version: 7.5.0
-:keycloak_upgrade_version: 15.0.2
-:project_versionDoc: 7.5
+:project_version_base: 7.6
+:project_version: 7.6.0
+:keycloak_upgrade_version: 18.0.x
```

Comment:
Yes, @mposolda.  You are right. When I published the Upgrade guide, I reset that at attribute to be 18.0.0.  That is the only time that attribute is used.

Reply:
@andymunro ah, ok. I missed that this PR was closed and replaced by some other.

It depends how hard it is to re-publish the documentation. If it is non-trivial effort to re-publish, it should be probably ok to wait for RH-SSO 7.6.1. Otherwise, it worth to update now.

Reply:
Actually, I have to republish the Release Notes because of a problem with operator upgrade, so I will apply your comments and republish the Release Notes.

Reply:
@andymunro Cool, Thanks!